tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -1092,6 +1092,91 @@ describe("Test Application deployment based on 'clusterSelector'", { tags: '@p1'
       )
     })
   )
+
+  qase(22,
+    it("Fleet-22: Test install app to new set of clusters from old set of clusters", { tags: '@fleet-22' }, () => {
+      const repoName = 'default-multiple-apps-cluster-selector'
+      const new_key = 'key_third_cluster'
+      const new_value = 'value_third_cluster'
+
+      gitRepoFile = 'assets/git-repo-multiple-app-cluster-selector.yaml'
+
+      cy.accesMenuSelection('Continuous Delivery', 'Clusters');
+      cy.contains('.title', 'Clusters').should('be.visible');
+
+      // Assign label to the clusters 
+      dsFirstTwoClusterList.forEach(
+        (dsCluster) => {
+          cy.assignClusterLabel(dsCluster, key, value);
+        }
+      )
+
+      // Create a GitRepo targeting cluster selector created from YAML.
+      cy.clickNavMenu(['Git Repos']);
+      cy.wait(500);
+      cy.clickButton('Add Repository');
+      cy.contains('Git Repo:').should('be.visible');
+      cy.clickButton('Edit as YAML');
+      cy.addYamlFile(gitRepoFile);
+      cy.clickButton('Create');
+      cy.checkGitRepoStatus('default-multiple-apps-cluster-selector', '2 / 2');
+
+      // Check first application status on both clusters.
+      dsFirstTwoClusterList.forEach((dsCluster) => {
+        cy.checkApplicationStatus(appName, dsCluster, 'All Namespaces');
+      })
+
+      // Add label to the third cluster
+      cy.accesMenuSelection('Continuous Delivery', 'Git Repos');
```

Comment:
Would `cy.accesMenuSelection('Continuous Delivery', 'Clusters');` work to avoid next line?

Reply:
And maybe avoid the check on the Cluster title as well?
It adds a lot of lines.

Reply:
I will give a try to it and update it. Thanks for the suggestion.

Reply:
I have updated it, thanks for the suggestion.